x-pack/plugins/observability_solution/synthetics/server/routes/default_alerts/default_alert_service.ts

```diff
@@ -37,6 +39,12 @@ export class DefaultAlertService {
   }
 
   async setupDefaultAlerts() {
+    this.settings = await savedObjectsAdapter.getSyntheticsDynamicSettings(this.soClient);
+    if ((this.settings?.defaultConnectors ?? []).length === 0) {
```

Comment:
I don't think we want this. I think the alerts should still appear in the observability alerts table even without connectors, and this would prevent this.